app/components/build-status-chart.js

```diff
@@ -0,0 +1,145 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+
+const invervalOverrides = {
+  day: {
+    subInterval: '1hour',
+  },
+  week: {
+    subInterval: '1day',
+    tooltipLabelFormat: '%A, %b %e',
+  },
+};
+
+export default Component.extend({
+  classNames: ['insights-odyssey'],
+  classNameBindings: ['isLoading:insights-odyssey--loading'],
+  private: false,
+
+  insights: service(),
+
+  intervalSettings: computed(function () {
+    return this.get('insights').getIntervalSettings(invervalOverrides);
+  }),
```

Comment:
Computed property without tracked properties will never be re-calculated. Is it expected?

Reply:
yeah, definitely, me too :) I thought so too, but thought it wouldn't hurt to draw an attention here, just in case ;)